sdk/python/requirements/py3.8-ci-requirements.txt

```diff
@@ -555,7 +551,7 @@ pandas==1.5.3
     #   great-expectations
     #   pandavro
     #   snowflake-connector-python
-pandavro==1.5.2
+pandavro==1.8.0
```

Comment:
These files probably need to be regenerated. `pandavro` should not be here.

Reply:
resolved